IPython/core/guarded_eval.py

```diff
@@ -637,6 +637,7 @@ def _list_methods(cls, source=None):
 
 dict_keys: Type[collections.abc.KeysView] = type({}.keys())
 method_descriptor: Any = type(list.copy)
+module = type(builtins)
```

Comment:
Hum, 

I missed that in the previous PR but those should likely be :

```
from types import MethodDescriptorType
from types import ModuleType
```

Not blocking so let's open as another issue.

Reply:
Oh, `MethodDescriptorType` is neat. I missed it being added in 3.7, but since we are not supporting 3.6 for some time now this is good idea indeed.

Reply:
I can take care of this.